views/includes/metaHead.njk

```diff
@@ -8,6 +8,7 @@
         environment: '{{ appData.environment }}',
         locale: '{{ locale }}',
         localePrefix: '{{ localePrefix }}',
+        sessionExpirySeconds: {{ appData.config.get('cookies.expiryMs') / 1000 }}
```

Comment:
Should we set this in seconds rather than needing a calculation in a template?

Reply:
Perhaps – will have to just invert the calculation in the session config instead.

Reply:
... done!